### DISCUSSION TITLE
如何让侧边栏及内容导航显示在两边

### DISCUSSION THREAD

**Discussion by ebdmn** [vuepress-theme-hope/vuepress-theme-hope#3711] [Q&A] [answered]
![1702044562321](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/47840704/350faba3-82c0-4bac-89e3-fa8dc2c07493)

**Comment by Mister-Hope** [comment, marked as answer]
换个主题

详见: https://github.com/orgs/vuepress-theme-hope/discussions/3742

**Reply by ebdmn** [reply]
漂亮

**Reply by Mister-Hope** [reply]
https://github.com/orgs/vuepress-theme-hope/discussions/3742
